packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1108,6 +1161,9 @@ export class ProjectService {
             throw new ForbiddenError();
         }
 
+        // This job is the job model we use to compile projects
+        // This is not the graphile Job id we use on scheduler
+        // TODO: remove this old job method and replace with scheduler log details
```

Comment:
How do these work together on this PR? The work is happening on the scheduler but it still updates the old job table?